tests/boards/native_posix/native_tasks/testcase.yaml

```diff
@@ -1,5 +1,5 @@
 tests:
-  boards.native_posix:
+  boards.native_posix.native_tasks:
     arch_whitelist: posix
   boards.native_posix_64:
```

Comment:
Any reason not to rename the one below to be consistent?

Reply:
It is a good remark. Do you mean to create name boards.native_posix_64.native_tasks?